tools/mmp/driver.cs

```diff
@@ -358,6 +367,7 @@ static void Main2 (string [] args)
 				{ "xamarin-framework-directory=", "The framework directory", v => { xm_framework_dir = v; }, true },
 				{ "xamarin-full-framework", "Used with --target-framework=4.5 to select XM 4.5 Target Framework", v => { IsUnifiedFullXamMacFramework = true; } },
 				{ "xamarin-system-framework", "Used with --target-framework=4.5 to select XM 4.5 Target Framework", v => { IsUnifiedFullSystemFramework = true; } },
+				{ "disable-partial-static-registrar", "Disable use of partial static registrar.", v => disable_partial_static_registrar = true },
```

Comment:
Maybe you can use the existing --registrar flag instead: if --registrar:dynamic, then don't use the partial static registrar.

i.e. only use the partial static registrar if no --registrar flag is specified.